### ISSUE TITLE
Android talkback does not work as intended

### ISSUE THREAD

**Issue by razmik-gharibyan** [pichillilorenzo/flutter_inappwebview#1789]
- [x] I have read the [Getting Started](https://inappwebview.dev/docs/intro/) section
- [x] I have already searched for the same problem

## Environment

| Technology      | Version       |
|-----------------| ------------- |
| Flutter version |  3.13.4             |
| Plugin version  |  6.0.0-beta.24+1             |
| Android version |   12.4            |
| iOS version     |   -            |
| macOS version   |    -           |
| Xcode version   |    -           |

Device information: <Xiaomi mi9>

## Description

Android talkback enabled
*User opens webview
*User taps on element inside webview
*Element is read out loud and selected
*Swiping detects next element and read out loud

**Current behavior:** 
Note: Works only when appbar of scaffold is provided.
*User opens webview
*User taps on element inside webview
*Element is not selected and not read out loud
*User holds down a few seconds on element and 8 times out of 10 element is selected and read out loud
*Swiping after that is working and next element is selected
*Tapping outside of webview
*Tap inside same element of webview
*Element is not selected again, have to longpress

## Steps to reproduce
Open blank project and add InAppWebView as home of material app. I gave initialUrl as google.com
run app, tap on element with android talkback enabled.
